fireatlas/FireRunDaskCoordinator.py

```diff
@@ -99,7 +99,8 @@ def job_fire_forward(eventual_results: Tuple[Delayed], region: Region, tst: Time
 
     large_fires = find_largefires(allfires.gdf)
     save_large_fires_nplist(allpixels, region, large_fires, tst)
-    save_large_fires_layers(allfires.gdf, region, large_fires, tst, ted)
+    save_large_fires_layers(allfires.gdf, region, large_fires, tst, ted, combine_all=False)
+    save_large_fires_layers(allfires.gdf, region, large_fires, tst, ted, combine_all=True)
```

Comment:
It would make more sense to me to call `save_large_fires_layers` once and write both files since merging the duplicated rows takes a non-trivial amount of time.